nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -178,7 +178,7 @@ StaticLayer::processMap(const nav_msgs::msg::OccupancyGrid & new_map)
       new_map.info.origin.position.x,
       new_map.info.origin.position.y,
       true);
-  } else if (size_x_ != size_x || size_y_ != size_y ||
+  } else if (size_x_ != size_x || size_y_ != size_y ||  // NOLINT
```

Comment:
What was the lint failure here?

Reply:
It said there needed to be a brace on the same line as the `else_if`